app/controllers/houses_controller.rb

```diff
@@ -0,0 +1,16 @@
+class HousesController < ApplicationController
+
+  def index
+    @house = House.all.ordered
+  end
+
+
+  def show 
+    @house = House.find(params[:id])
+    @bookings_list= @house.bookings.sort_by(&:day) if params[:order]=="asc" || !params[:order]
+    @bookings_list= @house.bookings.sort_by(&:day).reverse if params[:order]=="desc"
```

Comment:
Ezt érdemes lehet egy kicst átírni. Sokkal jobb lenne, ha az `if` -nek lenne egy egyértelmű `else` ága, mert a jelenlegi formában hibára tudunk futni akkor, ha pl params-ban beküldöm, hogy `{ order: 'IV.  Béla }` . Tudsz olyat csinálni Rails-ben, hogy az értékadást kiemeled az if-else elé. pl.:
```ruby
változó = if feltétel
    'érték #1'
  else
    'érték #2'
  end

Reply:
Bocsi, ehhez van még egy kommentem. Itt is használjunk légyszi `order` -t a `sort_by` helyett.
Csaláshoz linkelek doksit: https://apidock.com/rails/v5.2.3/ActiveRecord/QueryMethods/reverse_order ;)